tests/unit/gapic/securitycenter_v1/test_security_center.py

```diff
@@ -4817,7 +4817,7 @@ def test_set_finding_state_flattened():
         _, args, _ = call.mock_calls[0]
         assert args[0].name == "name_value"
         assert args[0].state == finding.Finding.State.ACTIVE
-        # # # # # # # assert args[0].start_time == timestamp_pb2.Timestamp(seconds=751)
+        # # # # # # # # # # assert args[0].start_time == timestamp_pb2.Timestamp(seconds=751)
```

Comment:
I don't understand this change -- is there some meaning to the number of repeated hashmarks?

Reply:
This was an owlbot replacement gone wrong. It's been fixed on the master branch (#151) so I think must have branched off an old version of `master`.